website/addons/onedrive/static/onedrive.css

```diff
@@ -0,0 +1,39 @@
+.onedrive-confirm-selection {
```

Comment:
I'm not certain this file is necessary -- styling is generally handled higher up the chain. Will investigate further later.